apps/vh/admin-web/admin-web.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   values:
     java:
       disableTraefikTls: true
-      image: sdshmctspublic.azurecr.io/vh/admin-web:staging-f282040-202305021737 # {"$imagepolicy": "flux-system:vh-admin-web"}
+      image: sdshmctspublic.azurecr.io/vh/admin-web:prod-5102554-202303261024 # {"$imagepolicy": "flux-system:vh-admin-web"}
```

Comment:
Do VH know about this?

Reply:
This should not change the effective image running on cluster. It is just renaming to follow standard naming :

- Base files like `apps/vh/admin-web/admin-web.yaml` always points to base image policy name( like `vh-admin-web` without any `-prod`/`-staging` prefixes)
- Prod only always points to base image policy ( like `vh-admin-web` without any `-prod`/`-staging` prefixes)
- I have whitelisted `stg.yaml` files (which are also usually not allowed to be whitelisted)

Reply:
Are u happy to approve it @benjamin-v1 ?